src/components/views/MapView.js

```diff
@@ -2,9 +2,34 @@ import React, {Component} from "react";
 import {Header} from "semantic-ui-react";
 import {Helmet} from "react-helmet";
 import {Map, TileLayer, Marker, Popup, Polyline, CircleMarker, Tooltip} from "react-leaflet";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import of 'CircleMarker'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1128)